### ISSUE TITLE
Navigating in JTree with left / right key behaves as up / down key execpt for the first child node

### ISSUE THREAD

**Issue by pmouawad** [weisJ/darklaf#228]
**Describe the bug**

Hello,
Another user reported an issue which I have myself faced many times without being able to understand what leads to it.

Here is his report:
-----------------------------------------------------------------------------------------------
hi, found another issue (mostly related) while using keyboard arrows to navigate through tree view, left and right arrows are behaving like up-down arrows. Should jump to the parent element but just goes one up within the same parent. This is not an issue with WIndows Classic "look and feel"

-----------------------------------------------------------------------------------------------

**To Reproduce**

On my side, after using JMeter for few minutes, the bug appears with Darklaf.
If you are on a element that has a parent and click on left row, expected behaviour is to fold the node but you just go to the previous node instead.

**Screenshots**
NA
**Additional Information:**
 - OS: Windows 10
 - OS Version: 10
- Darklaf Version: 2.5.4, 2.5.3

**Additional context**
NA

**Comment by weisJ** [comment]
I'll try to replicate the issue. Does it happen reliably after some minutes or only some times? Does it happen more often with deeper (higher nesting) tree nodes or are all nodes affected approximately the same?

**Comment by sumitgts** [comment]
Hello, This happens with a very simple nested JTree. Issue is only with left arrow and everytime

**Comment by weisJ** [comment]
Thank you for the additional information @sumitgts. Do you also experience the issue only after some time or is this always happening? If it only happens after some time how long does it usually take to observe the behaviour?

**Comment by sumitgts** [comment]
You should be able to reproduce it instantly/always

**Comment by weisJ** [comment]
@sumitgts could you try out the latest snapshot version (i.e. `com.github.weisj:darklaf-core:latest.integration`) and check whether the issue is still present?

**Comment by pmouawad** [comment]
Hello,
Sorry for late reply, I tried version 2.5.5 and issue persists.

Regards
